### PR TITLE
store comments in datastore database

### PR DESCRIPTION
**Pull request by GovernessS**
turn comments into entities and set them inside the database 

(sorry for the double commit - something went wrong on my end and I needed to add something)

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/input/Comments.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.input;
+
+/** Will be used in the next PR (just setting it up!) */
```
Comment (sbahaud) [inline portfolio/src/main/java/com/google/sps/input/Comments.java:17]: Update this comment to describe the class.
So, "Class that stores comment details."

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 2]
```diff
@@ -26,30 +28,37 @@
...
+
     comments.add(text);
 
+    Entity taskEntity = new Entity("Entry");
+    taskEntity.setProperty("Text", text);
+    taskEntity.setProperty("timestamp", timestamp);
+
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    datastore.put(taskEntity);
+
```
Comment (sbahaud) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:54]: Move the above block for writing to datastore to a separate method.

private void persistComments(String text, long timestamp) {...}

File: portfolio/src/main/java/com/google/sps/input/Comments.java (hunk 1 above)
Comment (GovernessS) [reply portfolio/src/main/java/com/google/sps/input/Comments.java:17]: Done!

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java (hunk 2 above)
Comment (GovernessS) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:54]: Done!

**Pull request merged**